src/cancelContentPlay.js

```diff
@@ -14,7 +14,7 @@ const cancelContentPlay = function(player) {
   }
 
   // Avoid content flash on non-iPad iOS
-  if (videojs.browser.IS_IOS) {
+  if ((videojs.browser.IS_IOS) && (!videojs.browser.IS_IPAD)) {
```

Comment:
I believe there is an IS_IPHONE option you can use instead https://github.com/videojs/video.js/blob/3a859f97d0647fc3d1ef6b2f4b755c71da8bef63/src/js/utils/browser.js#L23